pysyncthru/__init__.py

```diff
@@ -58,62 +64,89 @@ def __init__(
         """Initialize the the printer."""
         self.url = construct_url(ip)
         self._session = session
-        self.data = {}  # type: Dict[str, Any]
+        self.data_printer_status = {}  # type: Dict[str, Any]
+        self.data_counter_status = {}  # type: Dict[str, Any]
         self.connection_mode = connection_mode
 
     async def update(self) -> None:
         """
         Retrieve the data from the printer and caches is in the class
         Throws ValueError if host does not support SyncThru
         """
-        self.data = await self._current_data()
+        self.data_printer_status = await self._current_data(DataType.PRINTER)
+        self.data_counter_status = await self._current_data(DataType.COUNTER)
+
 
-    async def _current_data(self) -> Dict[str, Any]:
+    async def _current_data(self, datatype: Any) -> Dict[str, Any]:

```

Comment:
It might make sense to split this function into _current_data_home and _current_data_counter or sth similar